packages/frontend-api/src/Resources/config/graphql-types/Pagination/ConnectionDecorator.types.yml

```diff
@@ -0,0 +1,10 @@
+ConnectionDecorator:
+    type: object
```

Comment:
Why it is not relay-connection type? See this https://github.com/overblog/GraphQLBundle/blob/master/docs/definitions/relay/connection.md